src/engine/currencyEngine.js

```diff
@@ -378,9 +378,16 @@ export class CurrencyEngine {
     }
     const customFeeSetting = this.engineInfo.customFeeSettings[0]
     const customFeeAmount = customNetworkFee[customFeeSetting] || '0'
-    if (networkFeeOption === 'custom' && customFeeAmount !== '0') {
+    if (networkFeeOption === 'custom') {
+      const customFeeNumber = Number(customFeeAmount)
+      if (customFeeNumber < 1 || isNaN(customFeeNumber)) {
+        // TODO: Throw proper InvalidCustomFee error
+        const e = new Error(`1 satPerByte`)
+        e.name = 'ErrorBelowMinimumFee'
+        throw e
+      }
       // customNetworkFee is in sat/Bytes in need to be converted to sat/KB
-      return parseInt(customFeeAmount) * BYTES_TO_KB
```

Comment:
Concerning that this fix is to prevent a user from entering less than 1 unit of satoshi per kilobyte, and that the previous code state was parsing the custom fee as an integer, it might be a good idea to look into whether this is a solution to the reported issue.